flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -40,4 +36,14 @@ private FlinkConfigOptions() {
           .intType()
           .defaultValue(100)
           .withDescription("Sets max infer parallelism for source operator.");
+
+  public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_INFER_SOURCE_LOCALITY =
+      ConfigOptions.key("table.exec.iceberg.infer-source-locality")
```

Comment:
this is not a `infer`, which means calculating sth based on other information. We need a more accurate config name. I am thinking `expose-split-locality-info`

Reply:
Better. Updated.